src/app/pages/jails/jail-add/jail-add.component.ts

```diff
@@ -118,538 +130,661 @@ export class JailAddComponent implements OnInit {
       type: 'input',
       name: 'ip6_addr',
       placeholder: T('IPv6 Address'),
-      tooltip: T('Configures network or internet access for the jail.\
- Type the IPv6 address for VNET and shared IP jails. Single interface\
- format: <i>interface|ip-address/netmask</i>. Multiple interface format:\
- <i>interface|ip-address/netmask,interface|ip-address/netmask</i>.\
- Example: <b>re0|2001:0db8:85a3:0000:0000:8a2e:0370:7334/24</b>'),
+      tooltip: T('Configure IPv6 networking or internet access for the \
+                  jail. Enter the IPv6 address for <a \
+                  href="https://www.freebsd.org/cgi/man.cgi?query=vnet&apropos=0&sektion=0&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
+                  target="_blank">VNET(9)</a> and shared IP jails. \
+                  <br>Single interface format: <b>[interface|]\
+                  ip-address[/netmask]</b>. <br>\
+                  Example: <b>re0|fe80::/64</b> <br>\
+                  Multiple interface format: <b>[interface|]ip-address\
+                  [/netmask],[interface|]ip-address[/netmask]</b>.<br>\
+                  Example: <b>re1|2607:f0d0:1002:51:0000:0000:0000:0004,\
+                  re5|2001:db8:85a3::8a2e:370:7334/24</b>'),
     },
     {
       type: 'input',
       name: 'defaultrouter6',
-      placeholder: T('Default IPv6 Router'),
-      tooltip: T('Type <i>none</i> or a valid IP address. Setting this\
- property to anything other than <i>none</i> configures a default route\
- inside a <b>VNET</b> jail.'),
+      placeholder: T('Default IPv6 Route'),
+      tooltip: T('A valid IPv6 address to use as the default route. \
+                  <br>Enter <b>none</b> to configure the jail with \
+                  no IPv4 default route. <br>\
+                  <b>A jail without a default route will not be \
+                  able to access the network without additional \
+                  configuration.'),
     },
     {
       type: 'input',
       name: 'notes',
       placeholder: T('Note'),
-      tooltip: T('Optional. Add any notes about the jail here.'),
+      tooltip: T('Store notes about the jail.'),
     },
     {
       type: 'checkbox',
       name: 'boot',
       placeholder: T('Auto-start'),
-      tooltip: T('Check to auto-start this jail at system boot time. Jails\
- are started and stopped based on the <b>priority</b> value.'),
+      tooltip: T('Check to auto-start the jail at system boot time. \
+                  Jails are started and stopped based on iocage \
+                  priority. Set in the <b>priority</b> field under \
+                  <b>Custom Properties</b>.'),
```

Comment:
Building on above: "...based on the value of the priority field in <b>Custom Properties</b>."